Added Spring Security in Action, Second Edition

stjepanjurekovic opened the pull request:
Hi,

Stjepan from Manning here. Laurentiu just started a new MEAP for Spring Security in Action, Second Edition. Thank you for considering it.

Best,

stjepanjurekovic committed:
Added 2 new titles to Books

Hi,

Stjepan from Maning here. I thought these 2 latest Spring titles from our catalog might be a good match for your list. Thank you for considering it.

Best,

stjepanjurekovic committed:
Added Spring Microservices in Action, 2nd Ed

Hi,

Stjepan from Manning here. I thought this title might be a good match for your list. Thank you for considering it.

Best,

stjepanjurekovic committed:
Added Spring Security in Action, Second Edition

Hi,

Stjepan from Manning here. Laurentiu just started a nem MEAP for Spring Security in Action, Second Edition. Thank you for considering it.

Best,

stjepanjurekovic committed:
Added Java Pers. with Spring Data and Hibernate

Hi,

Stjepan here from Manning. I thought this title might be a good match for your list. Thank you for considering it.

Best,

ThomasVitale commented:
Hi @stjepanjurekovic, thanks for your contribution. Could you update the pull request to keep the alphabetical sorting? Also, it would be nice to keep one item for each book, so I would recommend replacing the old item with the new edition instead of adding it next to it. Thank you.

stjepanjurekovic committed:
Deleted old edition of Spring Sec. in Action

Hi Thomas,

As you requested, I've deleted the older edition.

Best,

stjepanjurekovic committed:
Updated my entry to fit the alphabetical order

Hi Thomas,

As requested, I've moved this item to match the alphabetical order.

Best,

ThomasVitale committed:
Merge branch 'main' into main

ThomasVitale commented:
@stjepanjurekovic thanks for your contribution. I have synced the branch with main and merged your PR.

stjepanjurekovic commented:
Thank you, Thomas.

Best,

On Thu, Mar 2, 2023 at 8:12 PM Thomas Vitale ***@***.***>
wrote:

> @stjepanjurekovic <https://github.com/stjepanjurekovic> thanks for your
> contribution. I have synced the branch with main and merged your PR.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ThomasVitale/awesome-spring/pull/25#issuecomment-1452420621>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANDIEMI6ZOOWDQSDZ3ZS2T3W2DWJTANCNFSM6AAAAAAUAHLELM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Stjepan Jureković
Manning Publications - Marketing
***@***.***
